packages/kubernetes-1.23/kubernetes-1.23.spec

```
@@ -103,6 +103,35 @@ Conflicts: (%{_cross_os}image-feature(no-fips) or %{_cross_os}kubelet-1.23-bin)
 %description -n %{_cross_os}kubelet-1.23-fips-bin
 %{summary}.
 
+%package -n %{_cross_os}kube-proxy-1.23
+Summary: Container cluster node proxy
+Requires: %{_cross_os}conntrack-tools
+Requires: %{_cross_os}containerd
+Requires: %{_cross_os}ecr-credential-provider-1.23
+Requires: %{_cross_os}aws-signing-helper
+Requires: %{_cross_os}kube-proxy-1.23(binaries)
```

Is there a scenario where it makes sense to install `kube-proxy` but not `kubelet`? If not, you could bind them together, and then you would only need one copy of the license files:
```suggestion
Requires: %{_cross_os}kubelet-1.23
Requires: %{_cross_os}kube-proxy-1.23(binaries)
```

You would also not need to list most of the other dependencies, as they'd be pulled in transitively by the `kubelet` dependency.

---

Yeah, that makes sense. I don't see it ever being installed without kubelet.